client/components/price/price-filter.js

```diff
@@ -0,0 +1,74 @@
+const frenchTouch = {
+  TTCOnly: false,
+  HTOnly: false,
+  withTTC: true,
+};
+
+const usTouch = {
+  TTCOnly: false,
+  HTOnly: true,
+  withTTC: false,
+};
+
+const deutchTouch = {
+  TTCOnly: true,
+  HTOnly: false,
+  withTTC: true,
+};
+
+const showTaxes = {
+  CA: usTouch,
+  WE: usTouch,
+  WS: usTouch,
+  QC: usTouch,
+  DE: deutchTouch,
+  FI: deutchTouch,
+  SN: deutchTouch,
+  CZ: frenchTouch,
+  ES: frenchTouch,
+  FR: frenchTouch,
+  GB: frenchTouch,
+  IE: frenchTouch,
+  IT: frenchTouch,
+  LT: frenchTouch,
+  MA: frenchTouch,
+  NL: frenchTouch,
+  PL: frenchTouch,
+  PT: frenchTouch,
+  TN: frenchTouch,
+};
+
+function priceFilter($translate) {
+  function format(price, country, frequency) {
+    const theCountry = country || 'FR';
+    const taxes = showTaxes[theCountry];
+
+    if (price.withTax.value !== 0) {
+      if (taxes.TTCOnly) {
+        return `<span class="bold">${price.withTax.text}</span>`;
```

Comment:
I suppose there is no other way that to create the `html` here

Reply:
Do you agree that: `<span class="bold">` === `<strong>`?

Reply:
In this case yes, otherwise `strong` has the semantic meaning that the element is more important than the remaining of the page